web-client/public/js/createNetwork.js

```diff
@@ -114,14 +135,21 @@ containing "-", "_", and alpha-numeric characters only`);
         $("#createNetworkFormContainer").remove();
         grnState.customWorkbook = {
             genes : {},
-            source : null
+            source : null,
+            sources : null
         };
     // get sources from database
         queryNetworkDatabase({type:"NetworkSource", info:null}).then(function (response) {
             $("#createNetworkQuestions-container").append(createHTMLforForm(Object.keys(response.sources)));
             grnState.customWorkbook.sources = response.sources;
             grnState.customWorkbook.source = Object.keys(response.sources).length === 1 ?
                 Object.keys(response.sources)[0] : null;
+            for (let source in response.sources) {
+                let i = source.indexOf(":");
+                let date = new Date(source.substring(i + 1));
+                date = getFormattedDateTime(date);
```

Comment:
Hmmm this is a little sketchy. Starting the `date` variable out as a `Date` and then replacing it with its string…JavaScript lets you do this but other languages won’t, and ideally we don’t get into the habit of doing this. If we create a new value, we should put that in a new variable

Reply:
I changed this as requested